webui/react/src/e2e/fixtures/dev.fixture.ts

```diff
@@ -1,22 +1,8 @@
-import { Page } from '@playwright/test';
-
 import { expect } from 'e2e/fixtures/global-fixtures';
 import { BaseComponent, CanBeParent } from 'e2e/models/common/base/BaseComponent';
 import { BasePage } from 'e2e/models/common/base/BasePage';
 
 export class DevFixture {
```

Comment:
dev fixture is gonna be repurposed for testing development, not using the dev commands on the UI